task/clair-scan/0.1/clair-scan.yaml

```diff
@@ -9,6 +9,8 @@ metadata:
     tekton.dev/tags: "appstudio, hacbs"
   name: clair-scan
 spec:
+  description: >-
+    Scans the image for vulnerabilities with clair
   params:
     - name: image-digest
       description: Image digest to scan
```

Comment:
can you explain this what? does something scan an image digest?

Reply:
It scans the image identified by this digest